tm2/pkg/crypto/keys/client/version_test.go

```diff
@@ -0,0 +1,48 @@
+package client
+
+import (
+	"testing"
+	"context"
+
+	"github.com/gnolang/gno/tm2/pkg/commands"
+	"github.com/gnolang/gno/tm2/pkg/version"
+	"github.com/gnolang/gno/tm2/pkg/testutils"
+	"github.com/stretchr/testify/require"
+)
+
+func TestVersionApp(t *testing.T) {
```

Comment:
What do you think about renaming this to `TestClient_Version`? or something similar